package.json

```diff
@@ -24,7 +24,7 @@
         "theme": "light"
     },
     "engines": {
-        "vscode": "^1.49.0"
+        "vscode": "^1.53.0"
```

Comment:
Unrelated, didn't want to create a new PR for this.
A user with this extension can end up in an experiment for native notebooks & they must be on the latest version of VS Code, hence the engine must be udpated.